views/includes/page-templates/add-trust.html

```diff
@@ -41,8 +41,14 @@ <h1 class="govuk-heading-xl govuk-!-margin-10">{{ title }}</h1>
       {% endif %}
 
       {% if reviewedTrustsRows.length > 0 %}
-        <h3 class="govuk-heading-m">What you have reviewed</h3>
+      <div class="govuk-summary-card">
+        <div class="govuk-summary-card__title-wrapper">
+          <h2 class="govuk-summary-card__title"> What you have reviewed </h2>
+        </div>
+        <div class="govuk-summary-card__content">
+          <dl class="govuk-summary-list">
         {{ govukSummaryList({ rows: reviewedTrustsRows }) }}
+        </dl></div></div>
```

Comment:
Can you sort out this indentation please, it's hurting my eyes a bit 😄 
each on separate lines:
```
    </dl>
  </div>
</div>
```

Reply:
done :)